src/States/MainMenuState.cpp

```diff
@@ -74,6 +74,13 @@ void MainMenuState::initialize()
 	mFileIoDialog.anchored(false);
 	mFileIoDialog.hide();
 
+    Font* tiny_font = Utility<FontManager>::get().font(constants::FONT_PRIMARY, constants::FONT_PRIMARY_NORMAL);
```

Comment:
First off, I realize this name was pre-existing.

Would prefer a more descriptive name of how the font is being used, rather than a presumed property of the style of the font (`tiny_font`). Actual characteristics of the font may change due to style updates, though how it is being used, or rather what it represents is not based on style. The name makes me think of text used in a footer, but is hard to know looking at just the code.

Reply:
This is left over from the very beginning of the development process. I'm not opposed to changing it though it's probably better noted and left for a different change set.